### PR TITLE
Support Rails 5.1

### PR DESCRIPTION
**Pull request by TylerRick**
(another option would be the https://github.com/goatapp/ar_after_transaction fork)

### CODE REVIEW

File: .ruby-gemset [hunk 1]
```diff
@@ -0,0 +1 @@
+ar_after_transaction
```
Comment (grosser) [inline .ruby-gemset]: add .ruby-gemset to your global .gitignore ?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 log
+vendor
```
Comment (grosser) [inline .gitignore]: what are you trying to ignore ?

File: .ruby-version [hunk 3]
```diff
@@ -0,0 +1 @@
+2.3.1
```
Comment (grosser) [inline .ruby-version]: it's supposed to run on any ruby version, so should not need pinning

File: .travis.yml [hunk 4]
```diff
@@ -3,7 +3,10 @@ rvm:
   - 2.0.0
   - 2.1.5
   - 2.2.2
+  - 2.3.1
 gemfile:
  - gemfiles/rails3.gemfile
  - gemfiles/rails4.gemfile
+ - gemfiles/rails5.gemfile
```
Comment (grosser) [inline .travis.yml]: for consistency that should be 3_2 / 4_2 / 5_0 then ... maybe 51 or 5.1

**Comment by grosser** [comment]
running travis on 2.4 / 2.5 might be good too, but that can be it's own PR

might need to add matrix excludes for rails 3 vs ruby 2.3

File: .gitignore (hunk 2 above)
Comment (TylerRick) [reply .gitignore]: This is new to me, but it looks like bundle is creating these now:
```
gemfiles/.bundle
vendor/bundle
```

File: .ruby-version (hunk 3 above)
Comment (TylerRick) [reply .ruby-version]: Makes sense. Removing file. (This was added by the original author, @chamnap, not me.)

File: .travis.yml (hunk 4 above)
Comment (TylerRick) [reply .travis.yml]: Sounds good. Renamed to 5.1. I'll let you rename the 3 and 4 ones, if you want.

**Comment by TylerRick** [comment]
I agree, I need this to run on 2.4. Added that and some excludes to travis matrix. We'll see if it passes :crossed_fingers: 

Replacing this PR with #20 since I don't actually have a way to push to `chamnap:master` and GitHub won't let me change the source branch (fail, GitHab; GitLab, on the other hand, does let you, I'm pretty sure).

**Pull request closed without merging**